Tiltfile

```diff
@@ -0,0 +1,102 @@
+# -*- mode: Python -*-
+
+# global settings
+settings = {
+    "allowed_contexts": [
+        "kind-gatekeeper",
+    ],
+}
+settings.update(read_json(
+    "tilt-settings.json",
+    default={},
+))
+
+allow_k8s_contexts(settings.get("allowed_contexts", []))
+
+if settings.get("trigger_mode", "auto").lower() == "manual":
+    trigger_mode(TRIGGER_MODE_MANUAL)
+
+LDFLAGS = "-X github.com/open-policy-agent/gatekeeper/pkg/version.Version=latest"
+
+TILT_DOCKERFILE = """
+FROM golang:1.17 as tilt-helper
+# Support live reloading with Tilt
+RUN wget --output-document /restart.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/restart.sh  && \
```

Comment:
looks like this org is deprecated. can we use https://github.com/tilt-dev/rerun-process-wrapper?

Reply:
Good catch